lib/src/pull_down_button.dart

```diff
@@ -509,7 +519,7 @@ Future<VoidCallback?> _showMenu<VoidCallback>({
   required double menuOffset,
   required ScrollController? scrollController,
 }) {
-  final navigator = Navigator.of(context);
+  navigator ??= Navigator.of(context);
```

Comment:
I don't like this reassignment here. Rather create a new local variable like this `final resolvedNavigator = navigator ?? Navigator.of(context);`

Reply:
Also see https://github.com/notDmDrl/pull_down_button/pull/49#issuecomment-2161438989